pilot/proxy/envoy/watcher.go

```diff
@@ -92,7 +92,7 @@ func (w *watcher) Run(ctx context.Context) {
 	}
 
 	go watchCerts(ctx, certDirs, watchFileEvents, defaultMinDelay, w.Reload)
-	go w.retrieveAZ(ctx, time.Duration(time.Second*10))
+	go w.retrieveAZ(ctx, time.Duration(time.Second*10), 10)
```

Comment:
Try every 30s so that there is a 5m window for this thing to work.

Reply:
Also, push these constants to some other place that is easy to spot.. AZAPIRetryInterval, AZAPIRetryAttempts or something like that..